spec/index.md

```diff
@@ -4198,12 +4198,10 @@ State after
 
 S with
     messages = Older_messages · Younger_messages
-    canister_status[A.canister_id] = Stopping [(M.origin, M.transferred_cycles)]
+    canister_status[A.canister_id] = Stopping [(M.origin, M.transferred_cycles, S.time[A.canister_id])]
```

Comment:
Do we need to be this precise and hardwire in the spec that expiration takes 5 mins? How about leaving it underspecified & adding a non-deterministic stopping time out transition?

Reply:
Sure we could define a constant for that, but I'd still define the value of the constant (otherwise it's unclear what you should expect as a developer).